src/main/kotlin/app/revanced/patches/music/layout/branding/icon/CustomBrandingIconPatch.kt

```diff
@@ -108,7 +119,84 @@ object CustomBrandingIconPatch : BaseResourcePatch(
+                            ResourceGroup(
+                                "drawable-hdpi",
+                                "action_bar_logo_release.png",
+                                "action_bar_logo.png",
+                                "logo_music.png", // 6.32 and earlier
+                                "ytm_logo.png", // 6.33 and later
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-large-hdpi",
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-large-mdpi",
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-large-xhdpi",
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-xlarge-hdpi",
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-xlarge-mdpi",
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-mdpi",
+                                "action_bar_logo.png",
+                                "logo_music.png", // 6.32 and earlier
+                                "ytm_logo.png", // 6.33 and later
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-xhdpi",
+                                "action_bar_logo.png",
+                                "logo_music.png", // 6.32 and earlier
+                                "ytm_logo.png", // 6.33 and later
+                                "record.png",
+                            ),
+
+                            ResourceGroup(
+                                "drawable-xxhdpi",
+                                "action_bar_logo.png",
+                                "logo_music.png", // 6.32 and earlier
+                                "ytm_logo.png", // 6.33 and later
+                                "record.png",
+                            ),
+
+
+                            ResourceGroup(
+                                "drawable-xxxhdpi",
+                                "action_bar_logo.png",
+                                "logo_music.png", // 6.32 and earlier
+                                "ytm_logo.png", // 6.33 and later
+                            ),
+                        ).forEach { resourceGroup ->
+                            context.copyResources("$resourcePath/resource", resourceGroup)
+                        }
+                    } catch (e: Exception) {
```

Comment:
Why do you use try catch in this case ?

Reply:
To avoid potential errors since some of them might not have some icons but it could be rewritten and I don't remember if it gave any error at all.

Reply:
I see, but what if the first group have error, for example. So the excution for other groups will be skipped -> patch didn't worked as intended. You shoud rewrite it

Reply:
True, feel free to rewrite it when you have time.

Reply:
Yea, but I don't have write access in inotia repos 🫠

Reply:
resolved in https://github.com/inotia00/revanced-patches/pull/55/commits/243ad46507fc5bedcea004a1e421f3d9461132d8